locales/locales.php

```diff
@@ -1662,6 +1680,20 @@ public function __construct() {
 		$pap->wp_locale = 'pap';
 		$pap->slug = 'pap';
 
+		$pap_aw = new GP_Locale();
+		$pap_aw->english_name = 'Papiamento (Aruba)';
+		$pap_aw->native_name = 'Papiamento';
+		$pap_aw->lang_code_iso_639_1 = 'pap';
+		$pap_aw->lang_code_iso_639_2 = 'pap';
+		$pap_aw->slug = 'pap-aw';
+
+		$pap_cw = new GP_Locale();
+		$pap_cw->english_name = 'Papiamento (Curaçao and Bonaire)';
+		$pap_cw->native_name = 'Papiamentu';
```

Comment:
The native name needs to include the country name as well otherwise you can't distinguish them in a language dropdown menu for example.